### ISSUE TITLE
emptyLinesUntilLevel stopped working

### ISSUE THREAD

**Issue by marcosvrs** [pascalre/vscode-yaml-sort#122]
Is it only for me, or the `emptyLinesUntilLevel` setting stopped working?
For some reason, it stands in my settings.json, but it doesn't have any effect on my files.

My workspace settings.json:
```json
{
    // docker-compose.yml
    "vscode-yaml-sort.customSortKeywords_1": [
        "version",
        "services",
        "container_name",
        "image",
        "build",
        "command",
        "environment",
        "cap_add",
        "network_mode",
        "user",
        "privileged",
        "env_file",
        "shm_size",
        "hostname",
        "ports",
        "volumes",
        "devices",
        "health_check",
        "depends_on",
        "labels",
        "extra_hosts",
        "restart"
    ],
    // homer
    "vscode-yaml-sort.customSortKeywords_2": [
        "name",
        "header",
        "title",
        "subtitle",
        "logo",
        "message",
        "links",
        "theme",
        "colors",
        "services",
        "subtitle",
        "icon",
        "items",
        "url",
        "target"
    ],
    "vscode-yaml-sort.emptyLinesUntilLevel": 2,
    "vscode-yaml-sort.quotingType": "\"",
    "vscode-yaml-sort.useAsFormatter": true,
    "vscode-yaml-sort.useCustomSortRecursively": true,
    "vscode-yaml-sort.useLeadingDashes": false,
    "vscode-yaml-sort.sortArrays": true
}
```

Simple file:
```yml
key:
  key2: value
  key3: value
key4: value
```

No spaces after the message “Keys resorted successfully”. I tried all the options (Format, Sort, Custom sort 1, 2).
![image](https://user-images.githubusercontent.com/10147574/220376282-decaabe3-0616-427b-a537-b1fd210f4b95.png)

**Comment by pascalre** [comment]
This is indeed a bug. I added a test case to address it.

**Comment by marcosvrs** [comment]
Perfect! :)

**Comment by pascalre** [comment]
I fixed this issue in 6.4.1. Please tell me if it works correct for you. If you like this extension and like my work, please consider leaving a review on the [marketplace](https://marketplace.visualstudio.com/items?itemName=PascalReitermann93.vscode-yaml-sort&ssr=false#review-details).
